aldryn_style/models.py

```diff
@@ -88,18 +95,17 @@ def inline_style(self):
 
     def clean(self):
         if self.additional_class_names:
-            additional_class_names = list(html_class.strip() for html_class in self.additional_class_names.split(','))
+            additional_class_names = list(
+                html_class.strip() for html_class in self.additional_class_names.split(','))
```

Comment:
I know this just modifies existing code, but why would we be calling `list()` on a generator instead of a plain list comprehension?